src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -136,9 +134,9 @@ function throwOnInvalidObjectType(returnFiber : Fiber, newChild : Object) {
     invariant(
       false,
       'Objects are not valid as a React child (found: %s).%s',
-      childrenString === '[object Object]' ?
+      newChild.toString() === '[object Object]' ?
```

Comment:
It's weird that we do this here but nowhere else.

If this is `Object.create(null)` this will throw so probably can't do this. Could do `Object.prototype.toString.call(newChild)`.